corehq/apps/campaign/const.py

```diff
@@ -0,0 +1,3 @@
+GAUGE_METRICS = [
+    'total_number_of_cases',
```

Comment:
Will this ever be displayed to the user such that the user can choose from a list of metrics or will we hard-code the gauges with specific metrics?

Reply:
> Will this ever be displayed to the user

+1 for the possibility to allow users to choose the type of metric, in which case these choices should have translated labels.

Reply:
Yes, this will be displayed to the user though that part would come in the PR that handles the display. For now just added one example to show what this would be.
Let me update the example though.

Reply:
Updated options with translatable strings.
Left a ToDo to add choices to model once metrics list is populated in following PRs
https://github.com/dimagi/commcare-hq/pull/35949/commits/8cf212fcba09f386fcbe9ad7a66769824936d91c